apps/web/pages/settings/organizations/members.tsx

```diff
@@ -8,6 +8,10 @@ export {
   type PageProps,
 } from "@calcom/features/ee/organizations/pages/settings/getServerSidePropsMembers";
 
+export const getLayout = (page: React.ReactElement) => (
+  <SettingsLayout containerClassName="lg:max-w-screen-lg">{page}</SettingsLayout>
+);
+
```

Comment:
Making this a bit wider on orgs when we have the space to do so really allows us to add more data in tables as they grow